rustls/src/crypto/aws_lc_rs/mod.rs

```diff
@@ -200,3 +217,25 @@ mod ring_shim {
 
 /// AEAD algorithm that is used by `mod ticketer`.
 pub(super) static TICKETER_AEAD: &ring_like::aead::Algorithm = &ring_like::aead::AES_256_GCM;
+
+/// Are we in FIPS mode?
+pub(super) fn fips() -> bool {
+    aws_lc_rs::try_fips_mode().is_ok()
+}
+
+#[cfg(test)]
+mod tests {
+    #[cfg(feature = "fips")]
+    #[test]
+    fn default_suites_are_fips() {
```

Comment:
Looks like in this commit, these tests would still fail?

Reply:
Double oops! Fixed that too.